sdk/keyvault/keyvault-certificates/review/keyvault-certificates.review.api.md

```diff
@@ -0,0 +1,286 @@
+    emailAddress?: string;
+    name?: string;
+    phone?: string;
+}
+
+// @public
+export interface Contacts {
+    contactList?: Contact[];
+    readonly id?: string;
+}
+
+// @public
+export interface DeletedCertificate extends Certificate {
+    readonly deletedDate?: Date;
+    recoveryId?: string;
+    readonly scheduledPurgeDate?: Date;
+}
+
+// @public
+export interface ErrorModel {
+    readonly code?: string;
+    readonly innerError?: ErrorModel;
+    readonly message?: string;
+}
+
+// @public
+export interface IssuerAttributes {
+    readonly created?: Date;
+    enabled?: boolean;
+    readonly updated?: Date;
+}
+
+// @public
+export interface IssuerCredentials {
+    accountId?: string;
+    password?: string;
+}
+
+// @public
+export interface IssuerParameters {
+    certificateTransparency?: boolean;
+    certificateType?: string;
+    name?: string;
+}
+
+// @public
+export interface KeyProperties {
+    curve?: JsonWebKeyCurveName;
+    exportable?: boolean;
+    keySize?: number;
+    keyType?: JsonWebKeyType;
+    reuseKey?: boolean;
+}
+
+// @public
+export interface KeyVaultClientGetCertificateIssuersOptionalParams extends coreHttp.RequestOptionsBase {
+    maxresults?: number;
+}
+
+// @public
+export interface KeyVaultClientGetDeletedCertificatesOptionalParams extends coreHttp.RequestOptionsBase {
+    includePending?: boolean;
+    maxresults?: number;
+}
+
+// @public
+export interface KeyVaultClientImportCertificateOptionalParams extends coreHttp.RequestOptionsBase {
+    certificateAttributes?: CertificateAttributes;
+    // Warning: (ae-forgotten-export) The symbol "CertificatePolicy" needs to be exported by the entry point index.d.ts
+    certificatePolicy?: CertificatePolicy_2;
+    password?: string;
+    tags?: {
+        [propertyName: string]: string;
+    };
+}
+
+// @public
+export interface KeyVaultClientSetCertificateIssuerOptionalParams extends coreHttp.RequestOptionsBase {
+    attributes?: IssuerAttributes;
+    credentials?: IssuerCredentials;
+    organizationDetails?: OrganizationDetails;
+}
+
+// @public
+export interface KeyVaultClientUpdateCertificateIssuerOptionalParams extends coreHttp.RequestOptionsBase {
+    attributes?: IssuerAttributes;
+    credentials?: IssuerCredentials;
+    organizationDetails?: OrganizationDetails;
+    provider?: string;
+}
+
+// @public
+export interface KeyVaultClientUpdateCertificateOptionalParams extends coreHttp.RequestOptionsBase {
+    certificateAttributes?: CertificateAttributes;
+    certificatePolicy?: CertificatePolicy_2;
+    tags?: {
+        [propertyName: string]: string;
+    };
+}
+
+// @public
+export interface LifetimeAction {
+    // Warning: (ae-forgotten-export) The symbol "Action" needs to be exported by the entry point index.d.ts
+    action?: Action;
+    // Warning: (ae-forgotten-export) The symbol "Trigger" needs to be exported by the entry point index.d.ts
+    trigger?: Trigger;
+}
+
+// @public
+export interface NewPipelineOptions {
```

Comment:
Replace with core-http PipelineOptions (see e.g. https://github.com/Azure/azure-sdk-for-js/pull/5711).

Reply:
Already done in #5669.

Reply:
Ahh thank you!